ocaml-lsp-server.opam

```diff
@@ -26,13 +24,20 @@ build: [
   [
     "dune"
     "build"
-    "-p"
-    name
     "-j"
     jobs
-    "@install"
-    "@runtest" {with-test}
-    "@doc" {with-doc}
+    "--root"
+    "."
+    "--ignore-promoted-rules"
+    "--no-config"
+    "--profile"
+    "release"
+    "%{name}%.install"
+  ]
```

Comment:
My intention with this is that this invocation is the same as `-p %{name}%` but allows to see other packages in the workspace.